### PR TITLE
Xonotic lan

### PR DESCRIPTION
**Pull request by jackharrhy**
make sure to review commands and dates and such, poster to soon follow!

### CODE REVIEW

File: content/events/2019/contents.lr [hunk 1]
```diff
@@ -6,3 +6,4 @@ body:
 
 - Tuesday, October 29th: [Code Night - Kattis Competition](./kattis_comp/)
 - Sunday, March 31st: [MUNCS + Chess.NL - March Rapid](./march_rapid/)
+- Friday, November 15th: [Game Night! - Xonotic LAN](./2019/xonotic_lan/)
```
Comment (nint8835) [inline content/events/2019/contents.lr]: Move lan to above kattis comp, so the events are in order

File: content/events/2019/xonotic_lan/contents.lr [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# copy the xonotic zip to said ~/games folder, and unzip it 
+cp ~csclub/games/xonotic-0.8.2.zip .
+unzip xonotic-0.8.2.zip
+
+# execute the linux startup script for xonotic
+cd Xonotic
+./xonotic-linux64-sdl
+
+# or, startup xonotic and directly connect to the server being used for the LAN
+./xonotic-linux64-sdl +connect cssbin
```
Comment (nint8835) [inline content/events/2019/xonotic_lan/contents.lr]: cssbin is not a valid hostname even within the cs lab. You need to use the full hostname, or instruct the users to configure their hosts file.

File: content/events/2019/xonotic_lan/contents.lr [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+If you want to use the desktops at the university to play Xonotic, there's a quicker way to download the game files.
+
+```bash
+# head to your home dir, and make a directory named 'games'
+cd ~
+mkdir games
+cd games
+
+# copy the xonotic zip to said ~/games folder, and unzip it 
+cp ~csclub/games/xonotic-0.8.2.zip .
```
Comment (nint8835) [inline content/events/2019/xonotic_lan/contents.lr]: Have we confirmed that this shorthand works on labnet?

File: content/events/2019/xonotic_lan/contents.lr [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+<br />
+
+#### Connecting to the LAN
+
+If you're using the desktops on campus, then you should be set.
+
+However, if you're using your own setup, make sure you are connected to the `@Memorial` wifi network, _not_ the `@Memorial-Guest` network; **you cannot access the local network through `@Memorial-Guest`!**
+
+The server will be running on campus, by the hostname of `cssbin`, which MUNs internal DNS should redirect you to if you're connected to the correct network, otherwise try `cssbin.cs.mun.ca`.
```
Comment (nint8835) [inline content/events/2019/xonotic_lan/contents.lr]: Hostname always needs to be the full one

File: content/events/2019/xonotic_lan/contents.lr [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+After following the instructions above to download Xonotic, just run the following command in the root directory of the game to connect to the server.
+
+```bash
+# within the ~/games/Xonotic folder
+./xonotic-linux64-sdl +connect cssbin.cs.mun.ca
+```
+
+**Reminder: the keyboards and mouses in the lab are a little sub-par for highspeed first person shooters, and there's also a significant lack of audio coming from said computers too, therefore if you have your own gear to bring with you, bring it!**
```
Comment (nint8835) [inline content/events/2019/xonotic_lan/contents.lr]: Plural of mouse is mice

File: content/events/contents.lr [hunk 6]
```diff
@@ -8,3 +8,4 @@ body:
 
 - Tuesday, October 29th: [Code Night - Kattis Competition](./2019/kattis_comp/)
 - Sunday, March 31st: [MUNCS + Chess.NL - March Rapid](./2019/march_rapid/)
+- Friday, November 15th: [Game Night! - Xonotic LAN](./2019/xonotic_lan/)
```
Comment (nint8835) [inline content/events/contents.lr]: Move lan to above kattis comp, so the events are in order

File: content/events/2019/xonotic_lan/contents.lr (hunk 3 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: yep

File: content/events/2019/xonotic_lan/contents.lr (hunk 2 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: tmux__Users_jack_Repos_bd_bln-ui-support-widget-library

tmux__Users_jack_Repos_bd_bln-ui-support-widget-library

tested that very command, and it did work on the lab PCs, as you can see by the screenshot it does mody the /etc/hosts file if you're logged into the desktops, but i'll denote the entire name so those that follow via their own laptop don't get lost

File: content/events/contents.lr (hunk 6 above)
Comment (jackharrhy) [reply content/events/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 5 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 4 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 2 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/contents.lr (hunk 1 above)
Comment (jackharrhy) [reply content/events/2019/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

Comment (jackharrhy) [reply content/events/2019/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 4 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 5 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/contents.lr (hunk 6 above)
Comment (jackharrhy) [reply content/events/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

File: content/events/2019/xonotic_lan/contents.lr (hunk 2 above)
Comment (jackharrhy) [reply content/events/2019/xonotic_lan/contents.lr]: 149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

**Pull request merged**